app/src/editor/shared/components/Map/Map.jsx

```diff
@@ -161,9 +162,7 @@ export default function Map(props: Props) {
     const mapCenter = [centerLat, centerLong]
     const tileUrl = getTileUrl(skin)
 
-    // https://github.com/facebook/flow/issues/2221
-    // $FlowFixMe Object.values() returns mixed[]
-    const markerArray: Array<Marker> = useMemo(() => Object.values(markers), [markers])
+    const markerArray: Array<Marker> = useMemo(() => ((Object.values(markers): any): Array<Marker>), [markers])
```

Comment:
I do note that `any` is basically `$FlowFixMe` in disguise

Reply:
ikr

They even documented it and called it "unsafe and not recommended" – https://flow.org/en/docs/types/casting/#toc-type-casting-through-any – what a shame. I wonder if there's a better way.

Reply:
Looks like `Object.values`, `.keys` and `.entries` are  problematic (?) for 3 years now! – https://github.com/facebook/flow/issues/2221. Well. I'll stick to `any`.